html-aam/index.html

```diff
@@ -11625,6 +11625,87 @@ <h4 id="att-indeterminate">`indeterminate [IDL]`</h4>
             </tr>
           </tbody>
         </table>
+        <h4 id="att-interestfor-attribute">`interestfor`</h4>
+        <table class="data" aria-labelledby="att-interestfor-attribute">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>
+                <span class="el-context"> `interestfor` </span>
+              </td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td>
+                <a data-cite="html/form-elements.html#the-button-element">`button`</a>;
+                <a data-cite="html/text-level-semantics.html#the-a-element">`a`</a>;
+                <a data-cite="html/image-maps.html#the-area-element">`area`</a>
+              </td>
+            </tr>
+            <tr>
+              <th>[[WAI-ARIA-1.2]]</th>
+              <td>
+                <p>A popover is <dfn>"plain"</dfn> if it contains only elements with a computed role of generic, text and images.</p>
+                <p>A popover is <dfn>"rich"</dfn> if it is not "plain".</p>
+                <p>If the `interestfor`-associated element is invalid, or is not a not a popover: no `aria-expanded` mapping</p>
+                <p>
+                  If the `interestfor`-associated element is an accessibility ancestor of the interest invoker:
+                  <a class="core-mapping" href="#ariaExpandedUndefined">`aria-expanded=undefined`</a>
+                </p>
+                <p>If the `interestfor`-associated element is a <a>"plain"</a> popover:
+                  <ul>
+                    <li><p>Set <a class="core-mapping" href="#ariaHiddenTrue">`aria-hidden="true"`</a> on the target popover.</p></li>
+                    <li><p>If no other <a data-cite="accname-1.2/#dfn-accessible-name">accessible name</a> is available, use the popover's inner text for
```

Comment:
Separate question: is there a matching PR to update the accname spec to reflect this change as well?

Reply:
this may fall under step 2.5 in accName which is meant to broadly call out 'other' host language features which could provide accNames.

ideally we shouldn't need to add this to that spec, as part of the intent of having specific AAM specs is to identify these host language details.

but leaving this comment unresolved in case this does warrant unique consideration.